jobs/tests/templates/bin/pre-start

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -eu
+exec 2>&1
+export LANG=en_US.UTF-8
+JOB=etcd-tls
+BIN=pre-start
+JOB_DIR=/var/vcap/jobs/tests/data/$JOB
+CERTS_DIR=/var/vcap/data/targets
+TMP_DIR=/var/vcap/data/tmp/$JOB
+
+# put cfssl in our path
+export PATH=$PATH:/var/vcap/packages/cfssl/bin
+
+echo "[$(date)] $BIN/$$: regenerating certificates..."
+rm -rf $TMP_DIR
+mkdir -p $CERTS_DIR $TMP_DIR
+pushd $TMP_DIR 2>/dev/null 2>&1
+
+echo "[$(date)] $BIN/$$: configuring ca parameters"
+cat > ca.json <<EOF
+{
+  "signing": {
+    "default": {
+      "expiry": "8760h"
+    },
+    "profiles": {
+      "etcd": {
+        "usages": ["signing", "key encipherment", "server auth", "client auth"],
+        "expiry": "8760h"
+      }
+    }
+  }
+}
+EOF
+cat ca.json
+echo
+
+gencert() {
+  cn=$1
+  alts=$2
+  dest=$3
+
+  cat > cert.json <<EOF
+{
+  "CN": "$cn",
+  "key": {
+    "algo": "rsa",
+    "size": 2048
+  }
+}
+EOF
+  chpst -u vcap:vcap \
+    cfssl gencert \
+    -ca=$JOB_DIR/tls/cert.pem \
+    -ca-key=$JOB_DIR/tls/key.pem \
+    -config=ca.json \
+    -hostname=$alts \
+    -profile=etcd \
+    cert.json | \
+      cfssljson -bare out
+  mkdir -p $CERTS_DIR/$dest
+  mv out.pem     $CERTS_DIR/$dest/cert.pem
+  mv out-key.pem $CERTS_DIR/$dest/key.pem
+  echo ">> $CERTS_DIR/$dest/cert.pem"
+  openssl x509 -text -in $CERTS_DIR/$dest/cert.pem
+  echo
+  chown -R vcap:vcap $CERTS_DIR/$dest/cert.pem
+  chown -R vcap:vcap $CERTS_DIR/$dest/key.pem
```

Comment:
Again, investigate the chown-vs-chpst

Reply:
Still need to investigate this. `chpst` for some reason isn't working for us.